src/sidebar/components/autocomplete-list.js

```diff
@@ -46,6 +64,7 @@ export default function AutocompleteList({
           }}
           {...props}
         >
+          {/* @ts-ignore TODO: fix the type for this callback */}
```

Comment:
What was the issue here?

Reply:
I think I fixed it by the time I finished the draft of this PR... I rather forgot. Best to remove now :)